hashicorp/waypoint
# Return pipeline run bundle from `UI_GetPipelineRun`

## Why the change?

This was an oversight on my part. We need the extra info for the UI.

## What’s the plan?

- [x] Update protos

## What does it look like?

Given I have run [this pipeline](https://github.com/jgwhite/waypoint-demos/blob/a8925b436fadb7ad1f28750a0da8ff7175ba78e6/pipelines/waypoint.hcl#L23-L31).

When I run:

```sh
$ ./contrib/waypoint-grpc/waypoint-grpc.sh UI_GetPipelineRun '
{
  "pipeline": {
    "owner": {
      "project": {
        "project": "pipelines"
      },
      "pipeline_name": "one-step"
    }
  },
  "sequence": 1
}
'
```

Then I see this:

```json
{
  "pipelineRunBundle": {
    "pipelineRun": {
      "id": "01H2AQSAABY33HNR8MS32V6JW5",
      "sequence": "1",
      "pipeline": {
        "id": "01H2AQRY5WCMXTP17DP7R9DPE5"
      },
      "jobs": [
        {
          "id": "01H2AQSAACRZGX35JGP5BSJD4V"
        }
      ],
      "state": "SUCCESS"
    },
    "queueTime": "2023-06-07T10:52:35.021352377Z",
    "application": {
      "application": "web",
      "project": "pipelines"
    },
    "dataSourceRef": {
      "git": {
        "commit": "a8925b436fadb7ad1f28750a0da8ff7175ba78e6",
        "timestamp": "2023-05-23T21:14:53Z",
        "commitMessage": "Add b\n"
      }
    }
  },
  "rootTreeNode": {
    // ...snip...
  }
}
```

## How do I test it?

Essentially follow the steps above, in other words:

1. Run a pipeline
2. Use `waypoint-grpc` to call `UI_GetPipelineRun`
3. Verify `pipelineRunBundle` looks correct
